README.md

```diff
@@ -1,24 +1,242 @@
+
+- **[Authentication, User can manage his recipies and foods]**
+- **[Guest User can navigate the App and read all ]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ER DIAGRAM-->
+## :card_index: Entity Relationship Diagram <a name="er-diagram"></a>
+![er-diagram](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_erd_2_members.png?raw=true)
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+### Prerequisites
+
+In order to run this project you need:
+- Visual Studio Code
+- Git
+- Ruby
+- PostgreSQL
+- Rails
+
+```
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+git clone git@github.com:samlorlah/recipe-app.git
+
+cd recipe-app 
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  bundle install
+```
+```sh
+  npm install
+```
+
+Create Database (Mandatory)
+
+```
+rails db:create
+```
+
+If you have made a migration then run this command
+
+```
+rails db:migrate
+```
+```
+rails db:seed
+```
+### Usage
+
+To run the project, execute the following command:
+
+
+```sh
+  rails server or rails s
+``` 
+
+Run tests
+```
+rspec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Muyiwa Fawole**
+
+- GitHub: [@samlorlah](https://github.com/samlorlah)
+- Twitter: [@samlorlah](https://twitter.com/samlorlah)
+- LinkedIn: [Oluwamuyiwa Fawole](https://linkedin.com/in/https://www.linkedin.com/in/muyiwa-fawole/)
+
+👤 **Shuhad Loofer**
+
+- GitHub: [@Shuhad786](https://github.com/Shuhad786)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/shuhad-loofer)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Use importmaps or esbuild for loading JavaScript]**
+- [ ] **[Build the API for recipes based on User entries]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank:
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
```

Comment:
- [ ] [OPTIONAL] Kindly endeavor to fill the FAQ section with at least 2 Q&A(s), or optionally remove this section.

Reply:
Removed